src/app/authentication/authentication-routing.module.ts

```diff
@@ -0,0 +1,15 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+
+import { LoginComponent } from './pages';
+
+const routes: Routes = [
+  { path: '', component: LoginComponent }
+];
+
+@NgModule({
+  imports: [RouterModule.forChild(routes)],
+  exports: [RouterModule]
+})
+
+export class AuthenticationRoutingModule { }
```

Comment:
@carryall Should we remove a space from `{ }` out?
Or this is the is from the guideline? 🤔

Reply:
@Lahphim It's the format used in the route file generated by Angular when I generate the project with `ng new <project-name>`, do you think we should keep it?

Reply:
@carryall OK we can keep it as we know that it comes from generating.